### DISCUSSION TITLE
QSelect - making selection and tabbing rapidly, focus bounces back to current control

### DISCUSSION THREAD

**Discussion by danh65** [quasarframework/quasar#17611] [General - Components / Directives / etc]
I have users who can type very fast and have discovered that a selection can be made in one select control and if tabbing too fast, the focus moves to the second control and then bounces back to the first control.

I've been able to reproduce it in a codepen project: https://codepen.io/danh65/pen/MWNGpgO?editors=1010

How can I get around this behavior?
Thanks, Dan

**Comment by stefanvanherwijnen** [comment]
This is probably caused by an `async` filter or add function which refocuses on the first input after adding the "Curious" value to it. I am not sure how to solve it.
